### PR TITLE
Syntax highlighting for html techniques

### PR DESCRIPTION
**Pull request by fstrr**
* update code to support syntax highlighting libraries

Also:
* fix typos
* fix / replace broken links
* remove references to XHTML and HTML5

### CODE REVIEW

**Review by Dubersnack — APPROVED** [review]

File: techniques/html/H100.html [hunk 1]
```diff
@@ -26,23 +24,19 @@ <h2>Description</h2>
...
+        <p>A secure website has an email and password based login form.</p>
+        <pre><code class="language-html">&lt;form method="post" action="login"&gt;
+  &lt;div&gt;
+    &lt;label for="email"&gt;Email&lt;/label&gt;
+    &lt;input autocomplete="email" id="email" type="email" ...&gt;
   &lt;/div&gt;
   &lt;div&gt;
-    &lt;label for=&quot;password&quot;&gt;Password&lt;/label&gt;
-    &lt;input id=&quot;password&quot; type=&quot;password&quot; autocomplete=&quot;current-password&quot; ... &gt;
+    &lt;label for="password&quo"&gt;Password&lt;/label&gt;
```
Comment (patrickhlauke) [inline techniques/html/H100.html:34]:
```suggestion
    &lt;label for="password"&gt;Password&lt;/label&gt;
```

**Comment by patrickhlauke** [comment]
@mbgower @alastc reviewed, made a few fixes, snuck in a few long-overdue changes (just in the html techniques folder, but likely should do same on all other html files ... will do in a separate PR), and validated all files. this should be good to go.

**Review by bruce-usab — APPROVED** [review]

**Comment by bruce-usab** [comment]
Discussed on call 1/12.  We noted need for Preview and that build errors need to be resolved.

**Comment by alastc** [comment]
I started, there's a few tricky conflicts and it won't build yet, so it's taking a while.

**Comment by alastc** [comment]
Merge conflicts resolved, build working, and preview looks good.

@mbgower Let's move this one across the board quickly before we get more merge conflicts!

**Review by alastc — APPROVED** [review]

**Comment by mbgower** [comment]
@alastc not easily finding a link to the preview; can you please post?

**Review by hidde — APPROVED** [review]

**Comment by mbgower** [comment]
@fstrr either you or @patrickhlauke was going to clean these conflicts up as part of whichever of your holistic PRs was regarded as the 'top gun'. Have you had a chance to discuss?

**Comment by fstrr** [comment]
@mbgower [I'm working on it](https://github.com/w3c/wcag/pull/3738#issuecomment-2016593941). Got a bunch of merge conflicts to deal with, but am hoping to get this finished and pushed up today.

**Comment by fstrr** [comment]
@mbgower @patrickhlauke [I merged our two branches](https://github.com/w3c/wcag/pull/3757/files) and I think everything's good. Definitely needs a good look to be sure.

**Comment by patrickhlauke** [comment]
superseded by https://github.com/w3c/wcag/pull/3757

**Pull request closed without merging**